examples/next/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "next-playground",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev -p 1702",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
It would be nice to have prod build preview in next too.

Reply:
that's what `start` does

Reply:
Did not know that. Thanks!